src/ui/qgsvectorlayerpropertiesbase.ui

```diff
@@ -1395,7 +1395,7 @@ border-radius: 2px;</string>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelDisplayNameInfo">
                    <property name="text">
-                    <string>The feature display name is used in identify results and attribute table's dual view list.</string>
+                    <string>The feature display name is used in identify results, locator searches and attribute table's dual view list.</string>
```

Comment:
bonus points if you change this to "...and THE attribute table's dual view list"